python/lsst/obs/decam/decamNullIsr.py

```diff
@@ -37,58 +39,35 @@ class DecamNullIsrConfig(pexConfig.Config):
         default="instcal",
     )
 
-## \addtogroup LSST_task_documentation
-## \{
-## \page DecamNullIsrTask
-## \ref DecamNullIsrTask_ "DecamNullIsrTask"
-## \copybrief DecamNullIsrTask
-## \}
-
 
 class DecamNullIsrTask(pipeBase.Task):
-    r"""!Load an "instcal" exposure as a post-ISR CCD exposure
-
-    @anchor DecamNullIsrTask_
-
-    @section pipe_tasks_decamNullIsr_Contents  Contents
-
-     - @ref pipe_tasks_decamNullIsr_Purpose
-     - @ref pipe_tasks_decamNullIsr_Initialize
-     - @ref pipe_tasks_decamNullIsr_IO
-     - @ref pipe_tasks_decamNullIsr_Config
-
-    @section pipe_tasks_decamNullIsr_Purpose  Description
+    """Load an "instcal" exposure as a post-ISR CCD exposure
 
     Load "instcal" exposures from the community pipeline as a post-ISR exposure,
     and optionally persist it as a `postISRCCD`.
 
     This is used to retarget the `isr` subtask in `ProcessCcdTask` when you prefer to use
     the community pipeline instead of the LSST software stack to perform ISR on DECam images.
-
-    @section pipe_tasks_decamNullIsr_Initialize  Task initialisation
-
-    @copydoc \_\_init\_\_
-
-    @section pipe_tasks_decamNullIsr_IO  Invoking the Task
-
-    The main method is `runDataRef`.
-
-    @section pipe_tasks_decamNullIsr_Config  Configuration parameters
-
-    See @ref DecamNullIsrConfig
     """
```

Comment:
I think the above should be lifted out into a Task Topic Type. You might need to ask @jonathansick for how to do that.

https://developer.lsst.io/stack/task-topic-type.html